jenkins/deploy.sh

```diff
@@ -56,11 +56,11 @@ echo "Plan to deploy ${FPATH}.jar to $SERVER_URL (ID:$SERVER_ID)"
 
 if [ "$SIGN_FILE" == true ]; then
     # No javadoc and sources jar is generated for shade artifact only. Use 'sql-plugin' instead
-    SQL_ART_ID=`mvn exec:exec -q -pl $SQL_PL -Dexec.executable=echo -Dexec.args='${project.artifactId}'`
-    SQL_ART_VER=`mvn exec:exec -q -pl $SQL_PL -Dexec.executable=echo -Dexec.args='${project.version}'`
+    SQL_ART_ID=`mvn help:evaluate -q -pl $SQL_PL -Dexpression=project.artifactId -DforceStdout`
+    SQL_ART_VER=`mvn help:evaluate -q -pl $SQL_PL -Dexpression=project.version -DforceStdout`
     JS_FPATH="${SQL_PL}/target/${SQL_ART_ID}-${SQL_ART_VER}"
     SRC_DOC_JARS="-Dsources=${JS_FPATH}-sources.jar -Djavadoc=${JS_FPATH}-javadoc.jar"
-    DEPLOY_CMD="mvn -B '-Pinclude-databricks,!snapshot-shims' gpg:sign-and-deploy-file -s jenkins/settings.xml -Dgpg.passphrase=$GPG_PASSPHRASE"
+    DEPLOY_CMD="mvn -B gpg:sign-and-deploy-file -s jenkins/settings.xml -Dgpg.passphrase=$GPG_PASSPHRASE"
 else
     DEPLOY_CMD="mvn -B '-Pinclude-databricks,!snapshot-shims' deploy:deploy-file -s jenkins/settings.xml"
```

Comment:
Should the profile settings be removed here as they were in the other case?

Reply:
> '-Pinclude-databricks,!snapshot-shims' has no impacts to the mvn deploy

That's basically my point.  If it has no impact on the deploy, why are we specifying it?

Reply:
'-Pinclude-databricks,!snapshot-shims' has no impacts to the mvn deploy --> I mean these option only has impacts during building time, when deploying, the jars has been built out. So it has no impact to deploy

Reply:
Also when we add '-Pinclude-databricks,!snapshot-shims' in gpg:sign-and-deploy-file, it said that the -P option NOT support.

Reply:
My main point here is that if we specify `'-Pinclude-databricks,!snapshot-shims'` during one of these code paths, someone is going to wonder why it's not specified in the other codepath.  I totally agree, at this point it should not be **building** the jars, just deploying them.  As you said:

> '-Pinclude-databricks,!snapshot-shims' has no impacts to the mvn deploy --> I mean these option only has works during building/testing time, when deploying, the jars has been built out. So it has no impact to deploy

If it has no impact on the deploy then we should remove those options from the deploy command.  That way one doesn't wonder why one path where we sign is not building for databricks but the path where we avoid signing isn't.  Neither path is trying to build the artifact, whereas the profile settings falsely imply we do need it to build per those settings.

Reply:
Make sense, better to remove both `-Pinclude-databricks,!snapshot-shims`  here.